MatrixSDK/Data/MXRoom.h

```diff
@@ -762,10 +762,11 @@ FOUNDATION_EXPORT NSString *const kMXRoomDidFlushDataNotification;
  Create a temporary message event for the room.
  
  @param eventId the event id. A globally unique string with kMXEventLocalEventIdPrefix prefix is defined when this param is nil.
+ @param eventTypeString the type of the event. @see MXEventType. 
  @param content the event content.
  @return the created event.
  */
-- (MXEvent*)fakeRoomMessageEventWithEventId:(NSString*)eventId andContent:(NSDictionary<NSString*, id>*)content;
+- (MXEvent*)fakeRoomMessageEventWithEventId:(NSString*)eventId eventType:(MXEventTypeString)eventType andContent:(NSDictionary<NSString*, id>*)content;
```

Comment:
I would rather create a new method named: ```- (MXEvent*)fakeEventWithEventId:(NSString*)eventId eventType:(MXEventTypeString)eventType andContent:(NSDictionary<NSString*, id>*)content;``` instead of changing the existing one